Solaris/MoonPhase.php

```diff
@@ -4,335 +4,485 @@
  * Adapted for PHP from Moontool for Windows (http://www.fourmilab.ch/moontoolw/)
  * by Samir Shah (http://rayofsolaris.net)
  * License: MIT
- **/
+ *
+ * Refactor by Jan Barasek <jan@barasek.com> http://baraja.cz
+ */
+
 namespace Solaris;
 
-class MoonPhase {
+class MoonPhase
+{
+
+	/**
+	 * @var int|null
+	 */
 	private $timestamp;
```

Comment:
Please change this and the other private variables to `protected` - see https://github.com/solarissmoke/php-moon-phase/issues/17